governance/SDK.md

```diff
@@ -0,0 +1,163 @@
+## SDK Community
+
+The community is organized as follows:
+
+- Every SDK has its own
+  [Github Team](https://github.com/orgs/cloudevents/teams), for example
+  `sdk-go-maintainers` is the group of mantainers for
+  [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
+- The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
+
+## New maintainers
+
+TBD
+
+## Ensuring projects health
+
+In order to ensure the health of the SDK projects, we define the following
+actions. These are crucial to show the commitment of the community to develop
+and support these libraries, which are an important part of the CloudEvents
+ecosystem.
+
+We define an SDK project _healthy_ if:
+
+1. It works with the latest version of the programming language
+2. It supports the latest versions of the integrated libraries/frameworks
+3. It receives security patches regularly
+4. It supports the last N-1 major versions of CloudEvents spec, as defined in
+   [Contribution Acceptance](../SDK.md#contribution-acceptance)
+5. Issues and PRs are triaged (labeled, commented, reviewed, etc) regularly
+
+We define a project `cloudevents/sdk-x` _not actively maintained_ if:
```

Comment:
What is an SDK has no activity because it doesn't need any work done on it? No language updates, no CE updates, no issues, etc... it's just working.

Reply:
Then It's fine, I don't define any actions in such cases. I define actions when issues/PR are not being triaged (archive/handover the project) or if the security patches are not applied and released.

Reply:
Maybe should i remove the criteria 6?

Reply:
Yep, number 6 is the one that got me thinking about this... I agree we need something, just not sure how to word it.
I wonder if instead of being too prescriptive, it should be a bit more vague to allow for the other sdk maintainers to use their best judgement.  Something like:
```
There has been no necessary activity within the SDK for at least 4 months. The other SDK maintainers will  evaluate what "no necessary activity" means, but often this could mean "no commits", or "no issue
discussions". However, if the SDK is stable and does not need to be update then it might be determined
that it current state is acceptable.
```
I don't know...

Reply:
I prefer to keep it as note more than as criteria, because it's ok to have it vague but it's too much open to "interpretations" to be a criteria. let me try to reword it